### PR TITLE
Mobile first

### PR DESCRIPTION
**Pull request by MasterLeiva**
Create Work and About Me sections
- Grid Usage
- Flexbox
- Github Flow
- Working Buttons
- Template Design
- Relative Sizes

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -51,5 +51,126 @@ <h2>
...
+          <li class="subcategories">Capybara</li>
+        </ul>
+        <h2 class="skills">  Skills</h2>
+        <ul>
+          <li class="subcategories">Database Management</li>
+          <li class="subcategories">Version Control</li>
+          <li class="subcategories">CLI</li>
+          <li class="subcategories">API Design</li>
+          <li class="subcategories">Web Development</li>
+        </ul>
```
Comment (shella12) [inline portfolio.html]: - [x] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, and each of them in a` li` tag as mentioned in the  [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

> Please use `ul `and `li` for categories in "About myself" section.` Languages`, `Frameworks`, and `Skills` should be list items.

 Example snippet:

```sh

<ul>
<li>Languages</li>
<li>Frameworks</li>
<li>Skills</li>
</ul>

```

File: portfolio.html [hunk 2]
```diff
@@ -51,5 +51,126 @@ <h2>
...
+        <div class="button-projects"><a>See Project</a></div>
+      </div>
+    </section>
+
+    <!-- About Myself section -->
+    <section class="about-myself">
+      <div class="about-me">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <div class="button-about-me"><a>Get my resume</a></div>
```
Comment (shella12) [inline portfolio.html]: - [ ]  Please implement the button interaction for hover and active state from the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=I98gQNqp5uJEZEvx-0) as it is the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) of the project:

> - Please Implement the button interactions (enable, hover, etc.).

Screenshot 2023-04-01 at 8 47 03 AM

File: portfolio.html
Comment (shella12) [inline portfolio.html:1]: - [ ] [OPTIONAL] I would highly suggest you to include an `index` page  (for example index.html) in the root directory or rename your `portfolio.html` page to `index.html`. 
Every published folder on your site needs an index page in order for your site's navigation to work properly. You can think of the index page as being the home or default page of your site and of any folders within your site. The index file is the file which is displayed when you enter your domain address in a web browser. The root is the directory which you first come to when using an [FTP client.](https://help.one.com/hc/en-us/articles/115005593805). SS of your project when I tried to open it. 


Screenshot 2023-04-01 at 8 34 24 AM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @MasterLeiva , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- Professional README.md ✅   
- Great design implementation ✅ 
- Clean code ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @MasterLeiva 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

💡 **[OPTIONAL]**

- [ ] To follow [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), let's not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

- [ ] Your `style.css` file

image

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**